packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -660,10 +665,12 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
               <div className={styles.sectionTitle}></div>
               <div className={styles.sectionField}>
                 <VisitFormExtensionSlot
-                  name="visit-form-top-slot"
+                  name="visit-form-bottom-slot"
                   patientUuid={patientUuid}
                   visitFormOpenedFrom={openedFrom}
-                  setVisitFormCallbacks={setVisitFormCallbacks}
+
+                  visitFormCallbacksRef={visitFormCallbacksRef} // Pass the ref instead
```

Comment:
Don't include comments that won't make sense to someone reading this code later. The whole point of comments is to make code _less_ confusing.